### PR TITLE
fix(responsiveness mobile): search breakpoint

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screen Shot 2021-06-05 at 11 08 01 AM
Screen Shot 2021-06-06 at 9 10 12 AM

**After:**
Screen Shot 2021-06-05 at 11 07 53 AM
Screen Shot 2021-06-06 at 9 10 19 AM

### CODE REVIEW

File: static/app/views/releases/detail/overview/chart/releaseChartControls.tsx [hunk 1]
```diff
@@ -151,13 +151,15 @@ const ReleaseChartControls = ({
...
+            size="sm"
+            title={t('This value includes only the current release.')}
+          />
+        </SectionHeading>
         <SectionValue key="total-value">{summary}</SectionValue>
-        <QuestionTooltip
-          position="top"
-          size="sm"
-          title={t('This value includes only the current release.')}
-        />
```
Comment (dashed) [inline static/app/views/releases/detail/overview/chart/releaseChartControls.tsx:160]: is this related to the breakpoint changes?

Comment (doralchan) [reply static/app/views/releases/detail/overview/chart/releaseChartControls.tsx:160]: Yeah it's cause I made changes to InlineContainer and we already had centering styles for a tooltip in SectionHeading. Will post screenshot

**Review by dashed — APPROVED** [review]
